apiserver/common/modelmanagerinterface.go

```diff
@@ -160,6 +162,7 @@ func (st modelManagerStateShim) GetModel(modelUUID string) (Model, func() bool,
 
 // Model implements ModelManagerBackend.
 func (st modelManagerStateShim) Model() (Model, error) {
+	st.State.Model()
```

Comment:
Maybe worth a comment why this line is needed?... Are we loading model into state here?